presto-client/src/main/java/com/facebook/presto/client/QueryResults.java

```diff
@@ -306,6 +308,8 @@ private static Object fixValue(TypeSignature signature, Object value)
             case INTERVAL_DAY_TO_SECOND:
             case DECIMAL:
                 return String.class.cast(value);
+            case CHAR:
+                return Strings.padEnd(String.class.cast(value), getCharTypeLength(signature), ' ');
```

Comment:
`String.class.cast` here is consistent with the rest of case code.

This string is what is returned in CLI or JDBC. Char data is sent without padding spaces to conserve bandwidth. Pad spaces are added on client side.